src/predict.rs

```diff
@@ -155,8 +155,8 @@ impl PredictionMode {
 
     let angle = match mode {
```

Comment:
Without the changes in `src/rdo.rs` the following assertion should be hit by `test_encode_decode::odd_size_frame_with_full_rdo_aom`.
```suggestion
    debug_assert!(mode != PredictionMode::UV_CFL_PRED || angle_delta == 0);
    let angle = match mode {
```

Reply:
Fixed in #2023.